sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala

```diff
@@ -363,11 +374,6 @@ object ViewHelper {
       analyzedPlan: LogicalPlan): Map[String, String] = {
     val queryOutput = analyzedPlan.schema.fieldNames
 
-    // Generate the query column names, throw an AnalysisException if there exists duplicate column
-    // names.
-    SchemaUtils.checkColumnNameDuplication(
-      queryOutput, "in the view definition", session.sessionState.conf.resolver)
```

Comment:
Why not make change here? Let checkColumnNameDuplication check the output of aliasedPlan?